servicebroker/test/routes/serviceinsance_test.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var supertest = require("supertest");
+var should = require('should');
+var uuid = require('uuid');
+
+var settings = JSON.parse(
+  fs.readFileSync(path.join(__dirname, '../../config/settings.json'), 'utf8'));
+var auth = new Buffer(settings.username + ":" + settings.password).toString('base64');
+describe('provisionService', function() {
```

Comment:
You need to truncate the table before each test.
Right now you are getting lucky based on execution order of this test and this test in sequence of the others.
Mocha is coming with a randomize support which we will run with eventually.